styles/styles.css

```diff
@@ -0,0 +1,137 @@
+/* basic colors, taken from coolors and spring at BMC picture:
+
+dark brown  - #2E231A
+    - use for most text? nav bar?
+    - maybe for some borders?
+dark rose - #723F43
+    - nav bar text color, flip on hover?
+    - emphasize certain words/links?
+dull pale blue - #7491AD
+    - background of sections?
+    - clicked links color? in rose section?
+dull pale rose - #8B6267
+    - background of sections?
+    - clicked links color in dull pale blue section?
+pale steel blue - #B8C4D1
+    - background of whole body?
+
+use BMC picture as header across top - about half of viewing height?
+
+use teacup picture as icon
+
+combo of Coraline page and french page - heading like coraline, then alternating color blocks like in french page
+
+Heading and navbar same across whole page, maybe just subheading changes?
+
+Index/Homepage:   
+    - very  basic intro, just that I am student currently studying to become a software developer via Ada
+    - 
+About:
+    - More in-depth about how I found coding, why I like it 
+Portfolio:
+    - list of current Ada projects on Github and brief descriptions of what they goals of each were, if they were pair programmed or not
+    - alternate blocks L and R with blank space inbetween?
+
+have nav bar that scrolls with page ** DO LAST**? Check old codepen stuff, probably have things there that can work. 
+    */
+
+.container {
+    width: 100vw;
+    height: 100vh;
+    display: grid;
+    grid-template: 1fr 56px 1fr .10fr/ .5fr 1fr 1fr .5fr;
+    
+}
+
+body {
+    background-color: #7491AD;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+header {
+    display: flex;
+    flex-direction: column;
+    background-color: #2E231A;
+    color: white;
+    grid-column: 1 / 5;
+    grid-row: 1 / 2;
+    background-image: url(../images/Spring\ at\ bryn\ mawr!\ 010.JPG);
+    background-size: 100%;
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-position-y: -100px;
+    text-align: center;
+    font-size: 3em;
+    text-shadow:
+		-2px -2px 0 #2E231A,
+		2px -2px 0 #2E231A,
+		-2px 2px 0 #2E231A,
+		2px 2px 0 #2E231A;
+    align-items: center;
+    justify-content: end;
+}
+
+.header-subtitle{
+    font-style: italic;
+    font-weight: lighter;
+}
+
+nav {
+    background-color: #2E231A;
+    grid-row: 2 / 3;
+    grid-column: 1 / 5;
+    text-align: center;
+    justify-content: center;
+    border-radius: 5px;
+}
+
+nav > ul > li {
+    display: inline-block;
+    align-content: center;
+    font-size: larger;
+}
+
+nav > ul > li > a {
+    color: #B8C4D1;
+    text-decoration: none;
+    padding: 10px 30px
+}
+
+nav > ul> li > a:hover {
+    background-color: #B8C4D1;
+    color: #2E231A;
+}
+
+nav > ul > li > a:visited:hover {
+    color: #723F43;
+}
+
+/* Set defaults for pages */
```

Comment:
I think I left it there specifically, as I planned to shuffle the main and footer portions of each page, but wanted the header and nav bar to be consistent across the site. So in my mind it was indicating that from that point on, things might be different page to page and this was the default, but everything above it would definitely be the same.